numpy/lib/function_base.py

```diff
@@ -338,6 +338,10 @@ def histogram(a, bins=10, range=None, normed=False, weights=None,
         if (mn > mx):
             raise AttributeError(
                 'max must be larger than min in range parameter.')
+        if not np.all(np.isfinite([mn, mx])):
+            raise AttributeError(
```

Comment:
That makes sense.  I'm happy to add that change here.  Should that be its own PR, instead?